tests/fast/IndexPrefetchDemo.toml

```diff
@@ -0,0 +1,6 @@
+[[test]]
```

Comment:
Can you integrate the new API to existing API tests,  such as 
https://github.com/apple/foundationdb/blob/14a63ee4d0928c7ddd1996a1b421e3a6a8117c1f/fdbserver/workloads/ApiCorrectness.actor.cpp#L300-L304 ?
The workload `ApiCorrectness*.actor.cpp` need to include your new api to make sure your API won't break ACID properties.

Reply:
Okay, I'll followup with another PR for API tests